### DISCUSSION TITLE
Wildcards in Suppression targetNames

### DISCUSSION THREAD

**Discussion by paulmccrady** [Azure/PSRule.Rules.Azure#2513] [Q&A] [answered]
When configuring a suppression in ps-rule.yaml and setting a targetName, is it possible to supply a wildcard?

**Comment by BernieWhite** [comment, marked as answer]
@paulmccrady No. With the `suppression:` key wildcards are not supported. You could do this with a suppression group.

See: https://github.com/Azure/PSRule.Rules.Azure/discussions/1999#discussioncomment-4743449

Instead of `endsWith` use `like` or `match`.

https://microsoft.github.io/PSRule/v2/concepts/PSRule/en-US/about_PSRule_Expressions/
https://microsoft.github.io/PSRule/v2/concepts/PSRule/en-US/about_PSRule_SuppressionGroups/

**Reply by paulmccrady** [reply]
@BernieWhite thankyou. Can you point me in the direction where it describes what 'name', 'type' etc actually relate to in the context of pre-flight and in-flight scans pls ?

**Reply by BernieWhite** [reply]
@paulmccrady Type or name will be the resource type and resource name. But specifically if you are using the `type:` or `name:` expression. The only supported value is `.` which refers to the current object. So: `type: .` or `name: .`

https://microsoft.github.io/PSRule/v2/concepts/PSRule/en-US/about_PSRule_Expressions/#type

See an example here: https://azure.github.io/PSRule.Rules.Azure/customization/permit-outbound-management/

**Reply by BernieWhite** [reply]
Also you may need to set binding options locally. https://azure.github.io/PSRule.Rules.Azure/customization/enforce-custom-tags/#binding-type
